userguide/sharing.rst

```diff
@@ -2620,3 +2476,142 @@ Return to
 |ui-options| on the desired file extent, then click :guilabel:`Edit`.
 Set the size to *0* as this causes the iSCSI target to use the new
 size of the file.
+
+
+.. index:: Time Machine
+.. _Creating Authenticated and Time Machine Shares:
+
+Creating Authenticated and Time Machine Shares
+----------------------------------------------
+
+macOS includes the
+`Time Machine <https://support.apple.com/en-us/HT201250>`__ feature
+which can be used to perform automatic back ups. %brand% supports
+Time Machine back ups for both :ref:`AFP <Apple (AFP) Shares>` and
+:ref:`SMB <Windows (SMB) Shares>` shares. The process for creating an
+authenticated share for a user is the same as creating a Time Machine
+share for that user.
+
+Before creating a Time Machine or authenticated share, go to
+:menuselection:`Storage --> Pools` to make a dataset for the share.
+For more information about dataset creation, refer to
+:ref:`Adding Datasets`.
+
+After creating the dataset for the guest share, go to
+:menuselection:`Storage --> Pools`,
+click the |ui-options| button for the dataset, then
+:guilabel:`Edit Permissions`.
+
+Enter the following information as shown in
+:numref:`Figure %s <creating_an_authenticated_share_fig>`.
+
+#. **ACL Type:** Select :guilabel:`Mac`.
+
+#. **User:** Use the drop-down to select the desired user account.
+   If the user does not yet exist on the %brand% system, go to
+   :menuselection:`Accounts --> Users`
+   to create one. Refer to
+   :ref:`users <Users>` for more information about creating a user.
+
+#. **Group:** Use the drop-down to select the desired group name.
+   If the group does not yet exist on the %brand% system, go to
+   :menuselection:`Accounts --> Groups`
+   to create one. Refer to
+   :ref:`groups <Groups>` for more information about creating a group.
+
+#. Click :guilabel:`SAVE`.
+
+
+To create an authenticated or Time Machine share:
+
+#. Go to :menuselection:`Sharing --> Apple (AFP) Shares`
```

Comment:
This should be in order of preference, and should mention why.  So SMB first, then maybe AFP with a warning.